### PR TITLE
Highlight colors refactor

### PR DESCRIPTION
**Pull request by Gregory711**
Created highlight color constants and used them to replace literals in code and in tests. Also added diagram to display what each highlight color is used for.

### CODE REVIEW

File: app/Components/Sudoku Board/Components/Cell.tsx [hunk 1]
```diff
@@ -325,27 +334,27 @@ const styles = (cellSize?: number, themeColor?: any) =>
...
       // styles for cells with isPeer prop
       color: "#000000",
-      backgroundColor: "#C5DDF4",
+      backgroundColor: PEER_SELECTED_COLOR,
     },
     sameValue: {
       // styles for cells with sameValue prop
       color: "#000000",
-      backgroundColor: "#c8dcc4",
+      backgroundColor: VALUE_SELECTED_COLOR,
```
Comment (Thomas-Gallant) [inline app/Components/Sudoku Board/Components/Cell.tsx:347]: These cells are not selected, can we maybe refactor this to 'TWIN_COLOR'? 
I am trying to figure out if there is an official name for this but can't seem to find anything.

Comment (Gregory711) [reply app/Components/Sudoku Board/Components/Cell.tsx:347]: Twin usually refers to exactly two of something but there could be more than two. What about CORRESPONDING_COLOR?

Comment (Thomas-Gallant) [reply app/Components/Sudoku Board/Components/Cell.tsx:347]: maybe IDENTICAL_VALUE_COLOR?

Comment (Gregory711) [reply app/Components/Sudoku Board/Components/Cell.tsx:347]: Sounds good

**Review by Thomas-Gallant — APPROVED** [review]

**Pull request merged**